pkg/remote/client.go

```diff
@@ -0,0 +1,144 @@
+package remote
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	"github.com/influxdata/influxdb-client-go/v2/api/write"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+const (
+	NodeIDTag  = "node_id"
+	ChainIDTag = "chain_id"
```

Comment:
[Questions] What is the usage of these constants?

Reply:
the keys for those tags

Reply:
Right! thanks